traitlets/config/tests/test_application.py

```diff
@@ -375,7 +372,7 @@ def test_aliases_multiple(self):
         # Test multiple > 2 aliases for the same argument
         class TestMultiAliasApp(Application):
             foo = Integer(config=True)
-            aliases = {("f", "bar", "qux"): "TestMultiAliasApp.foo"}
+            aliases = {("f", "bar", "qux"): "TestMultiAliasApp.foo"}  # type:ignore
```

Comment:
I think this might be fixed by https://github.com/ipython/traitlets/pull/810/commits/b805c9ed63330df126bc337f4b028b27def81c97#diff-5453d413070d06bcf7ba7f017b9cf364e774627fad69964a6471449e5c16dc37R361?